assets/javascripts/discourse/components/remind-assigns-frequency.js.es6

```diff
@@ -1,10 +1,19 @@
 import I18n from "I18n";
-import computed from "discourse-common/utils/decorators";
+import { computed } from "@ember/object";
+import discourseComputed from "discourse-common/utils/decorators";
 
 export default Ember.Component.extend({
-  selectedFrequency: null,
+  selectedFrequency: computed(
```

Comment:
That looks like a good candidate for the `@or` computed helper?

Reply:
Yes it crossed my mind while reading the code and thinking what I would change and forgot after :p will change :+1:

Reply:
done